Packs/Infoblox/Integrations/Infoblox/Infoblox.py

```diff
@@ -700,7 +699,11 @@ def transform_ip_context(ip_list: list[dict[str, Any]]) -> list[dict[str, Any]]:
     for ip in ip_list:
         i: dict[str, Any] = {}
         for k, v in ip.items():
-            i[IP_MAPPING[k]] = v
+            try:
+                i[IP_MAPPING[k]] = v
+            except KeyError:
+                demisto.debug(f"key '{k}' not found in expected IP context outputs. Setting it to output...")
+                i[string_to_context_key(k)] = v
```

Comment:
No need to get to an exception here just simple condition
```suggestion
            try:
                keys_transform = IP_MAPPING[k] if IP_MAPPING.get(k) else string_to_context_key(k)
                i[keys_transform] = v                
```

Reply:
https://github.com/demisto/content/pull/33772/commits/92328fe0ef436e615626613d52ecc4ab0b1376b2